apps/nova/src/hooks/call.ts

```diff
@@ -357,3 +357,56 @@ export function useCallEvents(
     mateVideo,
   };
 }
+
+export function useFullScreen() {
+  const intl = useFormatMessage();
+  const ref = useRef<HTMLDivElement>(null);
+  const [isFullScreen, setIsFullScreen] = useState(false);
+
+  const startFullScreen = useCallback(async () => {
+    try {
+      if (ref.current) {
+        setIsFullScreen(true);
+        await ref.current.requestFullscreen();
+      }
+    } catch (error) {
+      console.log(error);
+      toaster.error({
+        title: intl("error.unexpected"),
+      });
+    }
```

Comment:
```suggestion
   if (!ref.current) return 
    try {
        await ref.current.requestFullscreen();
    } catch (error) {
      toaster.error({
        title: intl("error.unexpected"),
      });
    }
```

It is better to se the `isFullScreen` using the `fullscreenchange` event. Here is an [example](https://github.com/litespace-org/litespace/blob/c45fc5f29549d49a4ede2a9b668f2cebe8bf25c7/packages/luna/src/components/VideoPlayer/video.ts#L179-L189) 

```ts
  const onFullScreen = useCallback(() => {
    // check if the `fullScreenElement` is the same as `ref` (example: compare element ids)
    const fullscreen = document.fullscreenElement == ref.current; // <--- sudo code 
    setFullScreen(fullscreen)
  }, []);

  useEffect(() => {
    document.addEventListener("fullscreenchange", onFullScreen);
    return () => {
      document.removeEventListener("fullscreenchange", onFullScreen);
    };
  }, [onFullScreen]);
```